app/code/Magento/Inventory/README.md

```diff
@@ -1,5 +1,21 @@
-# Inventory
+# Inventory module
 
-**Inventory** provides implementation for inventory management.
-See [concept documentation](https://github.com/magento-engcom/magento2/wiki/Technical-Vision.-Catalog-Inventory)
-for further information.
+The `Inventory` module is part of the MSI (Multi-Source Inventory) project,
+which replaces the legacy `CatalogInventory` module with new and expanded features and APIs for Inventory Management.  
+ 
+The [Inventory Management overview](https://devdocs.magento.com/guides/v2.3/inventory/index.html)
+describes the MSI project in more detail.
+
+All Inventory Management modules follow the 
+[Single Responsibility Principle](https://en.wikipedia.org/wiki/Single_responsibility_principle).
+[Inventory management architecture](https://devdocs.magento.com/guides/v2.3/inventory/architecture.html) 
+provides additional insight about the overall structure of these modules.
+
+## Installation details
+
+This module is installed as part of Magento Open Source. It cannot be deleted or disabled.
```

Comment:
This is not really correct for MSI modules, especially that all the MSI modules are disabled for merchants who make upgrade 2.1, 2.2 -> 2.3 + MSI